tests/admin_inlines/tests.py

```diff
@@ -114,7 +114,12 @@ def test_tabular_non_field_errors(self):
         }
         response = self.client.post(reverse('admin:admin_inlines_titlecollection_add'), data)
         # Here colspan is "4": two fields (title1 and title2), one hidden field and the delete checkbox.
-        self.assertContains(response, '<tr><td colspan="4"><ul class="errorlist nonfield"><li>The two titles must be the same</li></ul></td></tr>')
+        self.assertContains(
```

Comment:
We don't need the parenthesis in cases like this and if you have to break something into multiple lines, I'd break before 79:
```
self.assertContains(
    response,
    '<tr><td colspan="4"><ul class="errorlist nonfield">'
    '<li>The two titles must be the same</li></ul></td></tr>'
)
```